### PR TITLE
simple thymeleaf + non-rest controller

### PR DESCRIPTION
**Pull request by koics97**
-created new controller (TaskController)
-created simple thymeleaf template (ViewTasks.html)
-added thymeleaf dependecies

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,8 +55,17 @@
...
+		<dependency>
+			<groupId>org.thymeleaf</groupId>
+			<artifactId>thymeleaf-spring5</artifactId>
+			<version>3.1.2.RELEASE</version>
+		</dependency>
 	</dependencies>
-
 	<build>
 		<plugins>
 			<plugin>
```
Comment (deb05121) [inline pom.xml:71]: I don't know what happened here.

File: src/main/resources/templates/index.html
Comment (deb05121) [inline src/main/resources/templates/index.html:1]: I like it. I guess it turned out very good. I would like to try it.

File: src/main/resources/templates/ViewTasks.html
Comment (deb05121) [inline src/main/resources/templates/ViewTasks.html:1]: It's ok. I got it. I see you send a String to browser.

**Review by deb05121 — COMMENTED** [review]
I suppose you can merge this code.

**Review by deb05121 — APPROVED** [review]
That's ok.

File: pom.xml (hunk 1 above)
Comment (koics97) [reply pom.xml:71]: deleted empty line

**Pull request merged**